src/main/java/com/synopsys/integration/jenkins/detect/service/strategy/DetectAirGapJarStrategy.java

```diff
@@ -91,18 +91,18 @@ public SetupCallableImpl(JenkinsIntLogger logger, Map<String, String> environmen
             RemoteJavaService remoteJavaService = new RemoteJavaService(logger, remoteJdkHome, environmentVariables);
             String javaExecutablePath = remoteJavaService.calculateJavaExecutablePath();
 
-            logger.info("Detect configured: " + airGapJarPath);
+            logger.info("Detect AirGap jar configured: " + airGapJarPath);
```

Comment:
Changed for clarity, both in the code and for the customer

Reply:
👍